src/MeshlessFV/MeshlessFVTree.cpp

```diff
@@ -491,6 +491,8 @@ void MeshlessFVTree<ndim,ParticleType,TreeCell>::UpdateGradientMatrices
         // Make local copy of hmask for active particle
         hmask = mfv->types[activepart[j].ptype].hmask;
 
+        activepart[j].levelneib = 0;
+
```

Comment:
After merge, yes. You haven't pushed the commit yet though. I guess technically we don't have to loop over periodic and mpi ghosts, but the difference is so small that I don't care